src/EconomicStrategy/EconomicStrategyHelpers.ts

```diff
@@ -79,7 +79,9 @@ const isAboveMinBalanceLimit = async (config: Config, nextAccount: Address): Pro
  */
 const isProfitable = async (txRequest: ITxRequest, config: Config): Promise<boolean> => {
   const paymentModifier = await txRequest.claimPaymentModifier();
-  const claimingGasCost = await config.util.estimateGas({});
+  const claimingGasCost = await config.util.estimateGas({
+    to: txRequest.address
```

Comment:
Also you should specify the data so it knows which function to call. You can use https://github.com/ethereum-alarm-clock/eac.js-lib/blob/master/lib/txRequest/txRequest.js#L307

Also u need to multiply the amount of gas returned by estimateGas by the gasPrice to get the cost.

Reply:
Is the same gasPrice used to both claiming and execution?
https://github.com/ethereum-alarm-clock/eac.js-lib/blob/master/lib/txRequest/txRequest.js#L258

Reply:
No, the claiming will use the networkGasPrice. It's calculated somewhere in the overall claim process. You should use the same value here so there is no error due to the network changing its average between blocks (in reality this shouldn't be a huge deal but still)